README.md

```diff
@@ -3,3 +3,196 @@
 **This is not an official Google product.**
 
 Privacy data mapping and related plans management.
+
+## Set Up
+
+The central privacy team need to complete the steps below.
+
+### Workload Identity Federation
+
+Set up [Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation),
+and a service account with adequate condition and permission, see guide
+[here](https://github.com/google-github-actions/auth#setting-up-workload-identity-federation).
+
+    -  Service account used in Authenticating via Workload Identity Federation
+       needs [roles/storage.objectCreator] 
+       to snapshot the privacy mapping data/retention plans from GitHub to GCS.
+
+### GitHub Central Repository
+
+The GitHub central repository is the source of truth
+for privacy data mappings/wipeout plans.
+We rely on GitHub to preserve change history and enable multi-person review.
+
+The central privacy team can determine how to group privacy data mappings/wipeout plans as long as
+at least one level of group are needed
+(Sub folders in the root of the central GitHub repository are needed).
+Files containing privacy data mappings/wipeout plans can’t be stored
+directly in the root of the central GitHub repository.
+
+#### Privacy Data Mapping
+
+* Presubmit workflows for sanity checks
```

Comment:
I wonder if it's better to save the workflows as separate yaml files.

Reply:
done